### PR TITLE
move course-* components into course directory and ditch the prefix.

### PR DESCRIPTION
**Pull request by stopfstedt**
does the same with course visualisation components.

### CODE REVIEW

File: packages/ilios-common/app/styles/ilios-common/components/course/visualizations.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@use "../../mixins" as m;
+
+.course-visualizations {
```
Comment (jrjohnson) [inline packages/ilios-common/app/styles/ilios-common/components/course/visualizations.scss:3]: Is this a new consolidation? I couldn't find the matching file in the list, but make it's buried in deletes from other places?

File: packages/test-app/tests/integration/components/course/header-test.js [hunk 2]
```diff
@@ -3,10 +3,10 @@ import { setupRenderingTest } from 'test-app/tests/helpers';
 import { render } from '@ember/test-helpers';
 import { hbs } from 'ember-cli-htmlbars';
 import { a11yAudit } from 'ember-a11y-testing/test-support';
-import { component } from 'ilios-common/page-objects/components/course-header';
+import { component } from 'ilios-common/page-objects/components/course/header';
 import { setupMirage } from 'ember-cli-mirage/test-support';
 
-module('Integration | Component | course-header', function (hooks) {
+module('Integration | Component | course/header', function (hooks) {
```
Comment (jrjohnson) [inline packages/test-app/tests/integration/components/course/header-test.js:9]: Now you're just showing off! (would never have occurred to me to fix this too 😆)

**Review by jrjohnson — APPROVED** [review]

File: packages/ilios-common/app/styles/ilios-common/components/course/visualizations.scss (hunk 1 above)
Comment (stopfstedt) [reply packages/ilios-common/app/styles/ilios-common/components/course/visualizations.scss:3]: yes. was `packages/ilios-common/app/styles/ilios-common/components/course-visualizations.scss` before.

File: packages/ilios-common/addon/components/course/details.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { action } from '@ember/object';
 import scrollIntoView from 'scroll-into-view';
 import { service } from '@ember/service';
 
-export default class IliosCourseDetailsComponent extends Component {
```
[the remark this answers is not in the document]
Comment (stopfstedt) [reply packages/ilios-common/addon/components/course/details.js:6]: right!? we have less than a  handful `Ilios*` components left. can't remember either what compelled us to use this prefix in the first place.

**Pull request merged**